extensions/amp-story/0.1/amp-story-hint.css

```diff
@@ -17,6 +17,17 @@
 .i-amphtml-story-hint-container {
   transition-property: opacity !important;
   transition-duration: 200ms !important;
+  contain: strict !important;
```

Comment:
X-browser support is inconsistent: https://caniuse.com/#feat=css-containment

Reply:
Why not? Is this just nice-to-have?

Reply:
Correct